.github/workflows/cibuild.sh

```diff
@@ -5,9 +5,9 @@ set -ex
 PHASES=(${@:-CONFIGURE MAKE INSTALL CHECK DISTCHECK})
 COMPILER="${COMPILER:?}"
 COMPILER_VERSION="${COMPILER_VERSION}"
-CFLAGS=(-O1 -g)
-CXXFLAGS=(-O1 -g)
-LDFLAGS=()
+CFLAGS+=(-O1 -g)
+CXXFLAGS+=(-O1 -g)
+LDFLAGS+=()
```

Comment:
What the incremental operator `+=` does in this case (and in case the variable is not defined yet)? Would it be better to use CFLAGS="-O1 -g $CFLAGS"?